src/components/tooltip/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC } from 'react';
+
+import { Tooltip as ChakraTooltip, TooltipProps } from '@chakra-ui/react';
+
+const Tooltip: FC<TooltipProps> = ({ children, ...props }) => {
+  return (
+    <ChakraTooltip bg="gray.900" color="white" p="sm" hasArrow {...props}>
```

Comment:
Looking good!

One thing about the styles. What we do with the stylings is adding them into the themes.
Please move these styles in the corresponding component theme. 

Thank you!

Reply:
I think you can follow this theme to make the background work -> https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/components/tooltip.ts

Reply:
@GutierrezGo thanks for the tip, updated